articles/postgresql-aggregates-with-rust.md

```diff
@@ -5,7 +5,7 @@ Reaching for something like `SUM(vals)` or `AVG(vals)` is a common habit when us
 
 How do they work? What makes them different than a function? How do we make one? What kinds of other uses exist?
 
-We'll explore creating some basic ones using SQL, then create an extension that defines aggregates in Rust using [`pgx`][pgx] 0.3.0's new aggregate support.
+We'll explore creating some basic ones using SQL, then create an extension that defines aggregates in Rust using [`pgrx`][pgrx] 0.3.0's new aggregate support.
```

Comment:
I think this article should be left untouched and if we want to update this, it should be a full update to a latest version, unless we actually want to publish a 0.3.0

```suggestion
We'll explore creating some basic ones using SQL, then create an extension that defines aggregates in Rust using [`pgx`][pgx] 0.3.0's new aggregate support.
```